cookbook/case_studies/ml_training/spark_horovod/Dockerfile

```diff
@@ -42,6 +42,8 @@ RUN pip3 install wheel
 
 # Horovod-related installations
 RUN pip install tensorflow==2.6.0
+# Pre-requisite for horovod spark
+RUN apt install cmake
```

Comment:
@katrogan this is not required, look at line 17

Reply:
weird, this was failing for me until I added. I'll remove.